### ISSUE TITLE
NGS toolkit should provide an API to set cache configuration options

### ISSUE THREAD

**Issue by pditommaso** [ncbi/ngs#11]
Currently when accessing read collections by using the NGS API download reads are cached in the path `$HOME/ncbi/public` for open-access data, and the path $HOME/ncbi/dbGaP-xxxx for other.

There's no way to change these paths in the current API. It would be useful to have the ability to set a custom path for cached reads, including the possibility to disable the caching feature at all.

**Comment by kwrodarmer** [comment]
Yes, and for the record on this issue, we've already discussed that this option is being developed in an extension API. Will post progress and estimated delivery.
